### ISSUE TITLE
Allan Jacobs's Great Streets problem

### ISSUE THREAD

**Issue by majdal** [gboeing/osmnx#92]
Thanks for the beautiful (and beautifully written and maintained) library!

#### Problem description
I tried to run the very first basic example [here](https://github.com/gboeing/osmnx-examples/blob/master/notebooks/09-example-figure-ground.ipynb), and I get a 
`ValueError: not enough values to unpack (expected 4, got 0)` when running` fig, ax = ox.plot_figure_ground(point=point, filename=place, dpi=dpi)`. The full traceback is below.

#### What operating system, architecture, Python version, and OSMnx version are you using?
- OS: macOS 10.12.6
- Python: 3.6.1
- OSMnx: 0.6 (installed using miniconda)

#### Environment's packages
`conda list` output:
<details>
```
# packages in environment at /Users/majdal/miniconda3:
#
altair                    1.2.0                    py36_0    conda-forge
appnope                   0.1.0                    py36_0    conda-forge
asn1crypto                0.22.0                   py36_0
bleach                    2.0.0                    py36_0    conda-forge
branca                    0.2.0                      py_1    conda-forge
cffi                      1.10.0                   py36_0
click                     6.7                      py36_0    conda-forge
click-plugins             1.0.3                    py36_0    conda-forge
cligj                     0.4.0                    py36_0    conda-forge
conda                     4.3.27           py36hb556a21_0
conda-env                 2.6.0                h36134e3_0
cryptography              1.8.1                    py36_0
curl                      7.54.1                        0    conda-forge
cycler                    0.10.0                   py36_0    conda-forge
decorator                 4.1.2                    py36_0    conda-forge
descartes                 1.1.0                    py36_0    conda-forge
entrypoints               0.2.3                    py36_1    conda-forge
expat                     2.2.1                         0    conda-forge
fiona                     1.7.9                    py36_1    conda-forge
folium                    0.5.0                      py_0    conda-forge
freetype                  2.7                           1    conda-forge
freexl                    1.0.2                         2    conda-forge
gdal                      2.1.3                    py36_5    conda-forge
geopandas                 0.3.0                    py36_0    conda-forge
geopy                     1.11.0                   py36_0    conda-forge
geos                      3.5.1                         1    conda-forge
giflib                    5.1.4                         0    conda-forge
hdf4                      4.2.12                        0    conda-forge
hdf5                      1.8.18                        1    conda-forge
html5lib                  0.999999999              py36_0    conda-forge
icu                       58.1                          1    conda-forge
idna                      2.5                      py36_0
ipykernel                 4.6.1                    py36_0    conda-forge
ipython                   6.2.1                    py36_0    conda-forge
ipython_genutils          0.2.0                    py36_0    conda-forge
jedi                      0.10.2                   py36_0    conda-forge
jinja2                    2.9.6                    py36_0    conda-forge
jpeg                      9b                            1    conda-forge
json-c                    0.12.1                        0    conda-forge
jsonschema                2.6.0                    py36_0    conda-forge
jupyter_client            5.1.0                    py36_0    conda-forge
jupyter_core              4.3.0                    py36_0    conda-forge
kealib                    1.4.7                         2    conda-forge
krb5                      1.14.2                        0    conda-forge
libdap4                   3.18.3                        2    conda-forge
libgfortran               3.0.0                         0    conda-forge
libiconv                  1.14                          4    conda-forge
libnetcdf                 4.4.1.1                       6    conda-forge
libpng                    1.6.28                        1    conda-forge
libpq                     9.6.3                         0    conda-forge
libsodium                 1.0.10                        0    conda-forge
libspatialindex           1.8.5                         1    conda-forge
libspatialite             4.3.0a                       15    conda-forge
libssh2                   1.8.0                         1    conda-forge
libtiff                   4.0.7                         0    conda-forge
libxml2                   2.9.5                         0    conda-forge
markupsafe                1.0                      py36_0    conda-forge
matplotlib                2.0.2                    py36_2    conda-forge
mistune                   0.7.4                    py36_0    conda-forge
mkl                       2017.0.3                      0
munch                     2.2.0                    py36_0    conda-forge
nbconvert                 5.3.1                      py_1    conda-forge
nbformat                  4.4.0                    py36_0    conda-forge
networkx                  2.0                      py36_0    conda-forge
notebook                  5.1.0                    py36_0    conda-forge
numpy                     1.13.1                   py36_0
openjpeg                  2.2.0                         0    conda-forge
openssl                   1.0.2l                        0
osmnx                     0.6                      py36_0    conda-forge
packaging                 16.8                     py36_0
pandas                    0.20.3                   py36_1    conda-forge
pandoc                    1.19.2                        0    conda-forge
pandocfilters             1.4.1                    py36_0    conda-forge
pcre                      8.39                          0    conda-forge
pexpect                   4.2.1                    py36_0    conda-forge
pickleshare               0.7.4                    py36_0    conda-forge
pip                       9.0.1                    py36_1
proj4                     4.9.3                         4    conda-forge
prompt_toolkit            1.0.15                   py36_0    conda-forge
psycopg2                  2.7.3.1                  py36_0    conda-forge
ptyprocess                0.5.2                    py36_0    conda-forge
pycosat                   0.6.2                    py36_0
pycparser                 2.17                     py36_0
pygments                  2.2.0                    py36_0    conda-forge
pyopenssl                 17.0.0                   py36_0
pyparsing                 2.1.4                    py36_0
pyproj                    1.9.5.1                  py36_0    conda-forge
pysal                     1.14.2                   py36_1    conda-forge
python                    3.6.1                         2
python-dateutil           2.6.1                    py36_0    conda-forge
pytz                      2017.2                   py36_0    conda-forge
pyzmq                     16.0.2                   py36_2    conda-forge
readline                  6.2                           2
requests                  2.14.2                   py36_0
rtree                     0.8.3                    py36_0    conda-forge
ruamel_yaml               0.11.14                  py36_1
scipy                     0.19.1              np113py36_0
setuptools                27.2.0                   py36_0
shapely                   1.6.1                    py36_1    conda-forge
simplegeneric             0.8.1                    py36_0    conda-forge
six                       1.10.0                   py36_0
sqlalchemy                1.1.13                   py36_0    conda-forge
sqlite                    3.13.0                        0
terminado                 0.6                      py36_0    conda-forge
testpath                  0.3.1                    py36_0    conda-forge
tk                        8.5.18                        0
tornado                   4.5.2                    py36_0    conda-forge
traitlets                 4.3.2                    py36_0    conda-forge
vega                      0.4.4                    py36_1    conda-forge
vincent                   0.4.4                    py36_0    conda-forge
wcwidth                   0.1.7                    py36_0    conda-forge
webencodings              0.5                      py36_0    conda-forge
wheel                     0.29.0                   py36_0
xerces-c                  3.1.4                         3    conda-forge
xz                        5.2.2                         1
yaml                      0.1.6                         0
zeromq                    4.2.1                         1    conda-forge
zlib                      1.2.8                         3
```
</details>

#### Code that reproduces the issue
It's embarrassingly simple. I must be missing something?
```ipython
In [1]: import osmnx as ox
   ...: from IPython.display import Image
   ...:

In [2]: ox.config(log_file=True, log_console=True, use_cache=True)
   ...:
Configured osmnx

In [3]: img_folder = 'images'
   ...: extension = 'png'
   ...: size = 240
   ...: dpi = 40
   ...:

In [4]: place = 'sf'
   ...: point = (37.793897, -122.402189)
   ...: fig, ax = ox.plot_figure_ground(point=point, filename=place, dpi=dpi)
   ...: Image('{}/{}.{}'.format(img_folder, place, extension), height=size, width=size)
   ...:
Projected the GeoDataFrame "geometry to project" to UTM-10 in 0.01 seconds
Shell is not a LinearRing
Projected the GeoDataFrame "geometry to project" to EPSG 4326 in 0.01 seconds
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-4-261eab7acd7e> in <module>()
      1 place = 'sf'
      2 point = (37.793897, -122.402189)
----> 3 fig, ax = ox.plot_figure_ground(point=point, filename=place, dpi=dpi)
      4 Image('{}/{}.{}'.format(img_folder, place, extension), height=size, width=size)

~/miniconda3/lib/python3.6/site-packages/osmnx/plot.py in plot_figure_ground(G, address, point, dist, network_type, street_widths, default_width, fig_length, edge_color, bgcolor, smooth_joints, filename, file_format, show, save, close, dpi)
    833     elif point is not None:
    834         G = graph_from_point(point, distance=dist*multiplier, distance_type='bbox', network_type=network_type,
--> 835                              simplify=False, truncate_by_edge=True)
    836         G = simplify_graph(G, strict=False)
    837     else:

~/miniconda3/lib/python3.6/site-packages/osmnx/core.py in graph_from_point(center_point, distance, distance_type, network_type, simplify, retain_all, truncate_by_edge, name, timeout, memory, max_query_area_size, clean_periphery, infrastructure)
   1521     # create a bounding box from the center point and the distance in each
   1522     # direction
-> 1523     north, south, east, west = bbox_from_point(center_point, distance)
   1524
   1525     # create a graph from the bounding box

~/miniconda3/lib/python3.6/site-packages/osmnx/core.py in bbox_from_point(point, distance, project_utm)
   1357         # bounding coordinates
   1358         buffer_latlong, _ = project_geometry(buffer_proj, crs=crs_proj, to_latlong=True)
-> 1359         west, south, east, north = buffer_latlong.bounds
   1360         log('Created bounding box {} meters in each direction from {}: {},{},{},{}'.format(distance, point, north, south, east, west))
   1361

ValueError: not enough values to unpack (expected 4, got 0)
```

**Comment by gboeing** [comment]
I don't have a Mac computer to test your environment, but everything's working fine on my windows and linux machines with OSMnx 0.6, networkx 2.0, Shapely 1.6.1. Can you `conda update --all` to make sure every package's up to date?

For reference, your code snippet in one reproducible whole:

```python

import osmnx as ox
from IPython.display import Image
ox.config(log_file=True, log_console=True, use_cache=True)
img_folder = 'images'
extension = 'png'
size = 240
dpi = 40
place = 'sf'
point = (37.793897, -122.402189)
fig, ax = ox.plot_figure_ground(point=point, filename=place, dpi=dpi)
Image('{}/{}.{}'.format(img_folder, place, extension), height=size, width=size)
```

**Comment by gboeing** [comment]
@majdal just checking back...

**Comment by gboeing** [comment]
@majdal ok I'll close this for now but feel free to circle back if any additional questions

**Comment by yujinh** [comment]
I am having this problem also. I have done `conda update --all` but still get this error .

**Comment by gboeing** [comment]
@yujinh can you provide your full package list and environment info (ie, what OS, what python version, etc).

**Comment by yujinh** [comment]
macOS Sierra version 10.12.6

Python version 3.6

Package in environment:
`_ipyw_jlab_nb_ext_conf    0.1.0            py36h2fc01ae_0  
alabaster                 0.7.10           py36h174008c_0  
altair                    1.2.1                      py_0    conda-forge
anaconda                  custom           py36ha4fed55_0  
anaconda-client           1.6.5            py36h04cfe59_0  
anaconda-navigator        1.6.9            py36ha31b149_0  
anaconda-project          0.8.0            py36h99320b2_0  
appnope                   0.1.0            py36hf537a9a_0  
appscript                 1.0.1            py36h9e71e49_1  
asn1crypto                0.22.0           py36hb705621_1  
astroid                   1.5.3            py36h1333018_0  
astropy                   2.0.2            py36hf79c81d_4  
babel                     2.5.0            py36h9f161ff_0  
backports                 1.0              py36ha3c1827_1  
backports.shutil_get_terminal_size 1.0.0            py36hd7a2ee4_2  
beautifulsoup4            4.6.0            py36h72d3c9f_1  
bitarray                  0.8.1            py36h20fa61d_0  
bkcharts                  0.2              py36h073222e_0  
blaze                     0.11.3           py36h02e7a37_0  
bleach                    2.0.0            py36h8fcea71_0  
bokeh                     0.12.10          py36hfd5be35_0  
boto                      2.48.0           py36hdbc59ac_1  
bottleneck                1.2.1            py36hbd380ad_0  
branca                    0.2.0                      py_1    conda-forge
bzip2                     1.0.6                h92991f9_1  
ca-certificates           2017.08.26           ha1e5d58_0  
certifi                   2017.7.27.1      py36hd973bb6_0  
cffi                      1.10.0           py36h880867e_1  
chardet                   3.0.4            py36h96c241c_1  
click                     6.7              py36hec950be_0  
click-plugins             1.0.3                    py36_0  
cligj                     0.4.0                    py36_0  
cloudpickle               0.4.0            py36h13b7e56_0  
clyent                    1.2.2            py36hae3ad88_0  
colorama                  0.3.9            py36hd29a30c_0  
conda                     4.3.30           py36h173c244_0  
conda-build               3.0.27           py36hb78d8cd_0  
conda-env                 2.6.0                h36134e3_0  
conda-verify              2.0.0            py36he837df3_0  
contextlib2               0.5.5            py36hd66e5e7_0  
cryptography              2.0.3            py36h22d4226_1  
curl                      7.55.1               h7601780_3  
cycler                    0.10.0           py36hfc81398_0  
cython                    0.26.1           py36hd51f8eb_0  
cytoolz                   0.8.2            py36h290905f_0  
dask                      0.15.3           py36hc3ad2d6_0  
dask-core                 0.15.3           py36hc0be6b7_0  
datashape                 0.5.4            py36hfb22df8_0  
dbus                      1.10.22              h50d9ad6_0  
decorator                 4.1.2            py36h69a1b52_0  
descartes                 1.1.0                    py36_0    conda-forge
distributed               1.19.1           py36h4ae75d2_0  
docutils                  0.14             py36hbfde631_0  
entrypoints               0.2.3            py36hd81d71f_2  
et_xmlfile                1.0.1            py36h1315bdc_0  
expat                     2.2.4                h8f26bf8_1  
fastcache                 1.0.2            py36h8606a76_0  
filelock                  2.0.12           py36h0d0b4fb_0  
fiona                     1.7.9                    py36_1    conda-forge
flask                     0.12.2           py36h5658096_0  
flask-cors                3.0.3            py36h7387b97_0  
folium                    0.5.0                      py_0    conda-forge
freetype                  2.8                  h143eb01_0  
freexl                    1.0.2                         2    conda-forge
gdal                      2.1.3               np112py36_2    conda-forge
geopandas                 0.3.0                    py36_0    conda-forge
geopy                     1.11.0                   py36_0    conda-forge
geos                      3.5.1                         1    conda-forge
get_terminal_size         1.0.0                h7520d66_0  
gettext                   0.19.8.1             hb0f4f8b_2  
gevent                    1.2.2            py36ha70b9d6_0  
giflib                    5.1.4                         0    conda-forge
glib                      2.53.6               ha08cb78_1  
glob2                     0.5              py36h12393a9_0  
gmp                       6.1.2                h4a9834d_0  
gmpy2                     2.0.8            py36h7ef02cb_1  
greenlet                  0.4.12           py36hf09ba7b_0  
h5py                      2.7.0               np112py36_0  
hdf4                      4.2.12                        1  
hdf5                      1.8.17                        2  
heapdict                  1.0.0            py36h27a9ac6_0  
html5lib                  0.999999999      py36h79312fd_0  
icu                       58.2                 hea21ae5_0  
idna                      2.6              py36h8628d0a_1  
imageio                   2.2.0            py36h5e01289_0  
imagesize                 0.7.1            py36h3495948_0  
intel-openmp              2018.0.0             h68bdfb3_7  
ipykernel                 4.6.1            py36h3208c25_0  
ipython                   6.1.0            py36hf612aae_1  
ipython_genutils          0.2.0            py36h241746c_0  
ipywidgets                7.0.0            py36h24d3910_0  
isort                     4.2.15           py36hceb2a01_0  
itsdangerous              0.24             py36h49fbb8d_1  
jbig                      2.1                  h4d881f8_0  
jdcal                     1.3              py36h1986823_0  
jedi                      0.10.2           py36h6325097_0  
jinja2                    2.9.6            py36hde4beb4_1  
jpeg                      9b                   haccd157_1  
json-c                    0.12.1                        0    conda-forge
jsonschema                2.6.0            py36hb385e00_0  
jupyter                   1.0.0            py36h598a6cc_0  
jupyter_client            5.1.0            py36hf6c435f_0  
jupyter_console           5.2.0            py36hccf5b1c_1  
jupyter_core              4.3.0            py36h93810fe_0  
jupyterlab                0.27.0           py36hd3092eb_2  
jupyterlab_launcher       0.4.0            py36h93e02e9_0  
kealib                    1.4.6                         0  
lazy-object-proxy         1.3.1            py36h2fbbe47_0  
libcxx                    4.0.1                h579ed51_0  
libcxxabi                 4.0.1                hebd6815_0  
libdap4                   3.18.3                        2    conda-forge
libedit                   3.1                  hb4e282d_0  
libffi                    3.2.1                hd939716_3  
libgfortran               3.0.1                h93005f0_2  
libiconv                  1.15                 h99df5da_5  
libnetcdf                 4.4.1.1                       4    conda-forge
libpng                    1.6.32               hce72d48_2  
libpq                     9.5.4                         0  
libsodium                 1.0.13               hba5e272_2  
libspatialindex           1.8.5                         1    conda-forge
libspatialite             4.3.0a                       15    conda-forge
libssh2                   1.8.0                h1218725_2  
libtiff                   4.0.8                h8cd0352_9  
libxml2                   2.9.4                hbd0960b_5  
libxslt                   1.1.29               h95a2935_5  
llvmlite                  0.20.0                   py36_0  
locket                    0.2.0            py36hca03003_1  
lxml                      4.1.0            py36h8179fc0_0  
markupsafe                1.0              py36h3a1e703_1  
matplotlib                2.1.0            py36h5068139_0  
mccabe                    0.6.1            py36hdaeb55d_0  
mistune                   0.7.4            py36hccd6237_0  
mkl                       2017.0.3                      0  
mkl-service               1.1.2                    py36_3  
mpc                       1.0.3                hc455b36_4  
mpfr                      3.1.5                h7fa3772_1  
mpmath                    0.19             py36h9185fea_2  
msgpack-python            0.4.8            py36h46767b2_0  
multipledispatch          0.4.9            py36hc5f92b5_0  
munch                     2.2.0                    py36_0  
navigator-updater         0.1.0            py36h7aee5fb_0  
nbconvert                 5.3.1            py36h810822e_0  
nbformat                  4.4.0            py36h827af21_0  
ncurses                   6.0                  ha932d30_1  
networkx                  2.0              py36hefccab9_0  
nltk                      3.2.4            py36h27d1ea0_0  
nose                      1.3.7            py36h73fae2b_2  
notebook                  5.0.0            py36h462289e_2  
numba                     0.35.0              np112py36_0  
numexpr                   2.6.2               np112py36_0  
numpy                     1.13.3                    <pip>
numpy                     1.12.1                   py36_0  
numpydoc                  0.7.0            py36he54d08e_0  
odo                       0.5.1            py36hc1af34a_0  
olefile                   0.44             py36ha08bf50_0  
openjpeg                  2.2.0                h67e19a1_1  
openpyxl                  2.4.8            py36he899640_1  
openssl                   1.0.2l               h57f3a61_2  
osmnx                     0.6                      py36_0    conda-forge
packaging                 16.8             py36he5e8135_0  
pandas                    0.20.3           py36hd6655d8_2  
pandoc                    1.19.2.1             ha5e8f32_1  
pandocfilters             1.4.2            py36h3b0b094_1  
partd                     0.3.8            py36hf5c4cb8_0  
path.py                   10.3.1           py36hd33c240_0  
pathlib2                  2.3.0            py36h877a6d8_0  
patsy                     0.4.1            py36ha1b3fa5_0  
pcre                      8.41                 h29eefc5_0  
pep8                      1.7.0            py36hc268eb1_0  
pexpect                   4.2.1            py36h3eac828_0  
pickleshare               0.7.4            py36hf512f8e_0  
pillow                    4.2.1            py36h0263179_0  
pip                       9.0.1            py36hbd95645_3  
pkginfo                   1.4.1            py36h25bf955_0  
ply                       3.10             py36h10e714e_0  
proj4                     4.9.3                         4    conda-forge
prompt_toolkit            1.0.15           py36haeda067_0  
psutil                    5.4.0            py36ha052210_0  
psycopg2                  2.7.3.1          py36h1723d43_0  
ptyprocess                0.5.2            py36he6521c3_0  
py                        1.4.34           py36hecf431b_1  
pycodestyle               2.3.1            py36h83e8646_0  
pycosat                   0.6.2            py36h1486600_0  
pycparser                 2.18             py36h724b2fc_1  
pycrypto                  2.6.1            py36h72f2894_1  
pycurl                    7.43.0           py36hdb90038_3  
pyflakes                  1.6.0            py36hea45e83_0  
pygments                  2.2.0            py36h240cd3f_0  
pylint                    1.7.4            py36hdee9077_0  
pyodbc                    4.0.17           py36h5478161_0  
pyopenssl                 17.2.0           py36h5d7bf08_0  
pyparsing                 2.2.0            py36hb281f35_0  
pyproj                    1.9.5.1                  py36_0  
pyqt                      5.6.0            py36he5c6137_6  
pysal                     1.14.2           py36h48a81df_1  
pysocks                   1.6.7            py36hfa33cec_1  
pytables                  3.4.2               np112py36_0  
pytest                    3.2.1            py36h9963153_1  
python                    3.6.2                         0  
python-dateutil           2.6.1            py36h86d2abb_1  
python.app                2                py36h7fe2238_6  
pytz                      2017.2           py36h2e7dfbc_1  
pywavelets                0.5.2            py36h2710a04_0  
pyyaml                    3.12             py36h2ba1e63_1  
pyzmq                     16.0.2           py36h087ffad_2  
qt                        5.6.2                h9e3eb04_4    conda-forge
qtawesome                 0.4.4            py36h468c6fb_0  
qtconsole                 4.3.1            py36hd96c0ff_0  
qtpy                      1.3.1            py36h16bb863_0  
readline                  6.2                           2  
requests                  2.18.4           py36h4516966_1  
rope                      0.10.5           py36h5764ad1_0  
rtree                     0.8.3                    py36_0    conda-forge
ruamel_yaml               0.11.14          py36h9d7ade0_2  
scikit-image              0.13.0           py36h398857d_1  
scikit-learn              0.19.0              np112py36_0  
scipy                     0.19.1              np112py36_0  
seaborn                   0.8.0            py36h74df97e_0  
setuptools                36.5.0           py36h2134326_0  
shapely                   1.6.1                    py36_1    conda-forge
simplegeneric             0.8.1            py36he5b5b09_0  
singledispatch            3.4.0.3          py36hf20db9d_0  
sip                       4.18.1           py36h2824476_2  
six                       1.11.0           py36h0e22d5e_1  
snowballstemmer           1.2.1            py36h6c7b616_0  
sortedcollections         0.5.3            py36he9c3ed6_0  
sortedcontainers          1.5.7            py36ha982688_0  
sphinx                    1.6.3            py36hcd1b3e7_0  
sphinxcontrib             1.0              py36h9364dc8_1  
sphinxcontrib-websupport  1.0.1            py36h92f4a7a_1  
spyder                    3.2.4            py36h908396f_0  
sqlalchemy                1.1.13           py36h156b851_0  
sqlite                    3.13.0                        0  
statsmodels               0.8.0            py36h9c68fc9_0  
sympy                     1.1.1            py36h7f3cf04_0  
tblib                     1.3.2            py36hda67792_0  
terminado                 0.6              py36h656782e_0  
testpath                  0.3.1            py36h625a49b_0  
tk                        8.5.18                        0  
toolz                     0.8.2            py36h7b95164_0  
tornado                   4.5.2            py36h468dda9_0  
traitlets                 4.3.2            py36h65bd3ce_0  
typing                    3.6.2            py36haa2d9ef_0  
unicodecsv                0.14.1           py36he531d66_0  
unixodbc                  2.3.4                h4cb4dde_1  
urllib3                   1.22             py36h68b9469_0  
vega                      0.4.4                    py36_1    conda-forge
vincent                   0.4.4                    py36_0    conda-forge
wcwidth                   0.1.7            py36h8c6ec74_0  
webencodings              0.5.1            py36h3b9701d_1  
werkzeug                  0.12.2           py36h168efa1_0  
wheel                     0.29.0           py36h3597b6d_1  
widgetsnbextension        3.0.2            py36h91f43ea_1  
wrapt                     1.10.11          py36hc29e774_0  
xerces-c                  3.1.4                         0  
xlrd                      1.1.0            py36h336f4a2_1  
xlsxwriter                1.0.2            py36h3736301_0  
xlwings                   0.11.4           py36hc75f156_0  
xlwt                      1.2.0            py36h5ad1178_0  
xz                        5.2.3                ha24016e_1  
yaml                      0.1.7                hff548bb_1  
zeromq                    4.2.2                h131e0f7_1  
zict                      0.1.3            py36h71da714_0  
zlib                      1.2.11               h60db283_1  `

**Comment by Kuzcotopia** [comment]
I am also having issues with the basic demo code. I am using the "Getting started using python" guide you provided and after installing miniconda and running the basic code I am getting an error that says " shell is not a linear ring" 

<img width="1197" alt="screen shot 2017-11-07 at 9 41 47 am" src="https://user-images.githubusercontent.com/33330051/32509357-e9169936-c3a1-11e7-84af-b7626e45d63e.png">
 
I am running macOS Sierra version 10.12.6 and python version 3.6, also made sure to use conda update --all beforehand. 

Thanks, 
-Sam

**Comment by kuanb** [comment]
Drive by/quick response but I would check to make sure that the shape being returned is indeed a Polygon and not a LinearRing. If it is, you might want to convert it to a Shapely polygon and then use the `ox.graph_from_polygon` method instead:

![image](https://user-images.githubusercontent.com/6053396/32510335-226a8848-c3a5-11e7-9e6c-07fd6774fa13.png)

Then:
```
G = ox.graph_from_polygon(piedmont_shape, network_type='drive')
fig, ax = ox.plot_graph(ox.project_graph(G))
```

**Comment by gboeing** [comment]
@Kuzcotopia @yujinh @majdal this problem seems to be Mac specific, based on my tests on other platforms. And it appears to be a problem in one of OSMnx's dependencies, either geopandas or the GEOS library, as OSMnx isn't performing these operations natively. A similar problem and solution is referenced in this geopandas issue: https://github.com/geopandas/geopandas/issues/556 Have you seen that? Did that work for you?

**Comment by gboeing** [comment]
see also: https://github.com/geopandas/geopandas/issues/562

**Comment by abe732** [comment]
Same problem, the #556 issue you linked to didn't solve it

**Comment by utkarshshukla2912** [comment]
I got it solved by updating to shapely-1.6.4.post2 on MAC
`sudo pip3 install shapely -U`

**Comment by nathanielmhld** [comment]
I was getting a similar issue and solved it by rearranging to 
ox.graph_from_bbox(west, east, north, south)
as I found an example using it that way here: https://geoffboeing.com/2016/11/osmnx-python-street-networks/
I may have misaligned my variables, but at any rate, you may have done the same!
